### PR TITLE
AFRPAY-12236 v2 api update

### PR DESCRIPTION
**Pull request by ashrafhofney**

### CODE REVIEW

File: Helper/Refund.php [hunk 1]
```diff
@@ -29,8 +29,9 @@ public function createRefundRequest($order, $amount) {
             "shopConfig" => $this->config->getShopKey(),
             "refundAmount" => $amount,
             "refundCurrency" => $order->getOrderCurrencyCode(),
-            "description" => "Refund for order #".$order->getExtOrderId(),
+            "description" => "Refund for order #".$order->getData('purchaseId'),
```
Comment (cacevedojumee) [inline Helper/Refund.php:32]: please double check the order needed, either `merchantReferenceId` or `getRealOrderId()` but maybe is not the purchaseId right?

**Review by jmsakr — DISMISSED** [review]

File: view/frontend/web/css/jpay.css [hunk 2]
```diff
@@ -32,3 +31,78 @@
...
+    border-radius: 3px;
+    border-color: transparent;
+    float: right;
+    margin-right: 5px;
+}
+
+.payment-method.jpay .jpay-logo-ke {
+    height: 61px;
+    width: 200px;
+    background: no-repeat url('../images/secured_payfac_kenya.png');
```
Comment (cacevedojumee) [inline view/frontend/web/css/jpay.css:51]: Can we have a common pattern on country name ? ex 
`../images/secured_payfac_ke.png`

File: view/frontend/web/css/jpay.css [hunk 3]
```diff
@@ -32,3 +31,78 @@
...
+    border-radius: 3px;
+    border-color: transparent;
+    float: right;
+    margin-right: 5px;
+}
+
+.payment-method.jpay .jpay-logo-no-country {
+    height: 61px;
+    width: 200px;
+    background: no-repeat url('../images/secured_payfac_not country-specific.png');
```
Comment (cacevedojumee) [inline view/frontend/web/css/jpay.css:90]: Suggested file name. 
`'../images/secured_payfac_default.png`

**Review by cacevedojumee — CHANGES_REQUESTED** [review]

File: Helper/Refund.php (hunk 1 above)
Comment (ashrafhofney) [reply Helper/Refund.php:32]: this field is shown to the user when paying, we can keep current implementation as getRealOrderId will return a db id which won't be an interest for the user

**Review by cacevedojumee — APPROVED** [review]
looks good to me

**Review by magedghazouly — APPROVED** [review]

**Pull request merged**